<!DOCTYPE html>
<html>
  <head>
    <title>Integrative statistical analysis of -omics and GWAS data</title>
    <meta charset="utf-8">
    <meta name="author" content="Sina R√ºeger" />
    <meta name="date" content="2018-09-21" />
    <link href="libs/remark-css/default-fonts.css" rel="stylesheet" />
    <link href="libs/remark-css/default.css" rel="stylesheet" />
    <link href="libs/font-awesome/css/fontawesome-all.min.css" rel="stylesheet" />
    <link href="libs/ionicons/css/ionicons.min.css" rel="stylesheet" />
    <link rel="stylesheet" href="theme-public-defense.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: top, right, title-slide

# Integrative statistical analysis of -omics and GWAS data
## Public defense
### Sina R√ºeger
### 21 September 2018

---




&lt;!------- emoji üòÆ----------&gt;
&lt;!------- icons <i class="fab  fa-r-project " style="color:#88398a;"></i>----------&gt;
&lt;!------- icons <i class="ion  ion-heart "></i> ----------&gt;


---
class: inverse, left, middle

# Outline

- Introduction

- Results

- Discussion 

---
class: inverse, left, middle

# Outline

- Introduction (= explaining the title)

- Results (= what I did during the PhD)

- Discussion (= did it work? &amp; outlook)


&lt;!---------------- INTRODUCTION ---------------&gt;
---
class: inverse, center, middle

# Introduction

---
class: center, middle

## The goals to keep in mind: *Cornerstones of Medicine*

&lt;img src="img/cornerstones.png" width="400"&gt;


&lt;!---- Data analysis --------&gt;
---
class: center, middle

## 1. What is "data analysis" and "statistics"?

---
class: left, middle

### Summarise data

#### Example: Distribution of human height in 100 individuals

![](slides_files/figure-html/unnamed-chunk-1-1.png)&lt;!-- --&gt;



---
class: left, middle

### Summarise data

#### Example: Distribution of human height in 100 individuals

![](slides_files/figure-html/unnamed-chunk-2-1.png)&lt;!-- --&gt;


---
class: left, middle

### Identify relationships between A and B

#### Example: Relationship between age, sex and LDL (low-density lipoprotein)

.small[From: Freedman et al. (2004)]

&lt;img src="img/age_LDL.png" width="200"&gt; &lt;!----- &lt;img src="img/age_HDL.png" width="200"&gt; -------&gt;


&lt;!----------

---
class: left, middle

### Identifying patterns

#### Example: Genes mirror geography within Europe

&lt;img src="img/european-map.png" width="300"&gt;

.small[From: Novembre et al. (2008)]

---
class: left, middle

### Medical scores to assess patients

#### Example: APGAR score

Test done in newborns 1-5 minutes after birth. 

1. Skin color
2. Pulse rate
3. Reflex
4. Activity
5. Respiratory

&lt;img src="img/title-apgar.png" width="400"&gt;

.small[From: Apgar (1953)]

---------&gt;
---
class: left, middle

## What is the purpose of statistical methods? <i class="fas  fa-chart-line "></i>
&lt;!---------
1. By summarising data we can <i class="fas  fa-caret-down "></i> distill knowledge.

1. Helps us to understand the world and the complex systems we life in.

1. How data analysis works: 
    - identify a problem
    - define a question
    - collect data
    - answer the question (limited to the data used)
------&gt;

---
class: left, middle

## Central principles of data analysis

- Data represents oftentimes an **approximation** for something we cannot measure <i class="fas  fa-arrow-right "></i> *Use of BMI as a proxy for obesity*.

- The more data records we have, the **more precise** the estimation (of interest) <i class="fas  fa-arrow-right "></i> *Relationship between sample size and the standard error*.

&lt;!--- - Some questions are harder to answer than others: a **relationship between X and Y** is easier to quantify than showing that X is causing Y <i class="fas  fa-arrow-right "></i> * Correlation ‚â† Causation*.
  - it is **impossible to prove** a hypothesis, but we can reject a hypothesis <i class="fas  fa-arrow-right "></i> *aka hypothesis testing*. --------&gt;

---
class: left, middle

## Keep in mind <i class="fas  fa-hand-pointer "></i>

- Statistical methods: **summarise data** &amp; quantify relationships.

- Increasing **sample size** helps to estimate precisely. 

&lt;!---- Biology --------&gt;

---
class: center, middle

## 2. Bits of biology 

or: what is *-omics*?

---
class: left, middle

## DNA 

- is the blueprint of (human) life

- consists of nucleotides A, C, G and T 

---
class: left, middle

## Biological cascade


&lt;img src="img/cascade.png" width="800"&gt;

&lt;!--- 
- many genes
- basket of environment
- Dissecting the role of genetic and environmental effects. 

---&gt;

---
class: left, middle

## SNP (Single Nucleotide Polymorphism)

&lt;img src="img/snps-001.png" width="600"&gt;

---
class: left, middle

## SNP (Single Nucleotide Polymorphism)

&lt;img src="img/snps-002.png" width="600"&gt;


---
class: left, middle

.pull-left[
&lt;blockquote class="twitter-tweet" data-lang="en"&gt;&lt;p lang="en" dir="ltr"&gt;Two of my favourite things put together! &lt;a href="https://t.co/1jE0hWr0Yt"&gt;pic.twitter.com/1jE0hWr0Yt&lt;/a&gt;&lt;/p&gt;&amp;mdash; Mike Inouye (@minouye271) &lt;a href="https://twitter.com/minouye271/status/993073495205527552?ref_src=twsrc%5Etfw"&gt;May 6, 2018&lt;/a&gt;&lt;/blockquote&gt;
&lt;script async src="https://platform.twitter.com/widgets.js" charset="utf-8"&gt;&lt;/script&gt;
]

.pull-right[
## Linkage Disequilibrium (LD)

Parts of the **genome tend to be inherited together**, which creates sets of correlated SNPs in close proximity (‚Äòin linkage disequilibrium (LD)‚Äô).

]
&lt;!---------  ------&gt;




---
class: left, middle

## How DNA is measured

&lt;img src="img/genome.png" width="700"&gt;



---
class: left, middle

## Keep in mind <i class="fas  fa-hand-pointer "></i>

- We are only interested in the varying part of the genome <i class="fas  fa-arrow-right "></i> SNPs

- Parts of the genome are correlated <i class="fas  fa-arrow-right "></i> LD

&lt;!---- GWAS  --------&gt;
&lt;!--- combine DA and biology --------&gt;
---
class: center, middle

## 3. GWAS (genome-wide association study)

---
class: left, middle

## Relationship between a genotype and trait/disease

![](slides_files/figure-html/unnamed-chunk-3-1.png)&lt;!-- --&gt;

---
class: left, middle

## Relationship between a genotype and trait/disease

![](slides_files/figure-html/unnamed-chunk-4-1.png)&lt;!-- --&gt;

---
class: left, middle

## Relationship between a genotype and trait/disease
&lt;img src="img/gwas-a.png" width="600"&gt;


---
class: left, middle

## Relationship between MANY genotypes and trait/disease
&lt;img src="img/gwas-b.png" width="600"&gt;

---
class: left, middle

## GWAS summary statistics

&lt;img src="img/gwas-ss.png" width="400"&gt;

---
class: left, middle

## Large sample sizes needed

.pull-left[
&lt;img src="img/maf_effectsize.png" width="400"&gt;
]

.pull-right[
- Solution \#1: Exploit genetic correlation and use chips (cheap) instead of sequencing (expensive)

- Solution \#2: Form a consortium
]

---
class: left, middle

## Consortium of cohorts around the world

&lt;img src="img/consortium.png" width="700"&gt;
&lt;!---------
---
class: left, middle

## Example: GWAS in MDD

- Wray et al. (2018)

- 44 novel loci associated with major depression disorder (MDD)

- a meta-analysis of GWAS summary statistics from `\(135'458\)` cases and `\(344'901\)` from six cohorts. 

Resulting summary statistic 
- was integrated with gene expression data to identify the somatic tissues involved in MDD
- used to estimate heritability explained by the new findings
- to identify biological pathways relevant to MDD
- to investigate the relationships with genetically correlated traits
-----&gt;

---
class: left, middle

## Keep in mind <i class="fas  fa-hand-pointer "></i>

- Large sample sizes needed: **consortium**!

- Use of GWAS summary statistics with **follow-up methods**.  



&lt;!---- Missing data --------&gt;
---
class: center, middle

## 4. Missing data

---
class: left, middle

## How to infer data that is missing?

&lt;img src="img/map-arosa.png" width="650"&gt;


---
class: left, middle

## How to infer data that is missing?

&lt;img src="img/map-arosa-plain.png" width="650"&gt;

---
class: left, middle

## Basic principle


&lt;img src="img/genome_imputation.png" width="600"&gt;

&lt;!---------
---
class: left, middle

## Public reference panel

&lt;img src="img/map-1000genomes-populations.png" width="600"&gt;
-----------&gt;

&lt;!---- SSIMP --------&gt;
---
class: center, middle

## 5. Summary statistic imputation (SSIMP)

---
class: left, middle

## GWAS

&lt;img src="img/ssimp-a.png" width="700"&gt;

---
class: left, middle
## Untyped variant

&lt;img src="img/ssimp-b.png" width="700"&gt;

---
class: left, middle

## Genotype imputation

&lt;img src="img/ssimp-c.png" width="700"&gt;

---
class: left, middle

## Genotype imputation

&lt;img src="img/ssimp-d.png" width="700"&gt;

---
class: left, middle

## Summary statistic imputation

&lt;img src="img/ssimp-e.png" width="700"&gt;

---
class: left, middle

## Main equation

&lt;img src="img/ssimp-equation.png" width="700"&gt;


&lt;!---------------- RESULTS ---------------&gt;
---
class: inverse, center, middle

# Results


---
class: left, middle
# Output


1. <i class="fas  fa-arrow-right "></i> Evaluation and application of summary statistic imputation <i class="fas  fa-arrow-left "></i>
    - &lt;small&gt;*Evaluation and application of summary statistic imputation to discover new height- associated loci*  (R√ºeger, McDaid, and Kutalik 2018)&lt;/small&gt;

1. Improving summary statistic imputation for mixed populations
    - &lt;small&gt;*Improved imputation of summary statistics for mixed populations*  (R√ºeger, McDaid, and Kutalik 2017, not peer-reviewed)&lt;/small&gt;

1. Applications of summary statistic imputation
    - &lt;small&gt;*Rare and low-frequency coding variants alter human adult height* (Marouli et al. 2017)&lt;/small&gt;
    - &lt;small&gt;*Bayesian association scan reveals loci associated with human lifespan and linked biomarkers* (McDaid et al. 2017)&lt;/small&gt;

1. Contributions to research in infectious diseases
    - &lt;small&gt;*Impact of common risk factors of fibrosis progression in chronic hepatitis C* (R√ºeger et al. 2015)&lt;/small&gt;
    - ...

1. Minor contributions to other publications

---
class: left, middle

&lt;img src="img/rueeger-etal-2018-fp.png" width="800"&gt;

---
class: left, middle

# Goals

1. **Improve** imputation quality &amp; investigate parameters of SSIMP.

1. **Compare** summary statistic imputation to genotype imputation.

1. **Test** the utility of summary statistic imputation on a real case on human height data.


---
class: left, middle

# How to test new imputation methods
--

.pull-left[
.big[üôà]
]

---
class: left, middle

# 1. Compare SSIMP to genotype imputation

&lt;img src="img/results-ukbb-schema.png" width="600"&gt;

---
class: left, middle

&lt;img src="img/results-ukbb-associated.png" width="800"&gt;

---
class: left, middle

&lt;img src="img/results-ukbb-null.png" width="800"&gt;


---
class: left, middle

# 2. Test SSIMP on height GWAS

&lt;img src="img/results-exome-schema.png" width="600"&gt;

---
class: left, middle


## Locuszoom plot of results


&lt;img src="img/results-exome-locuszoom.png" width="500"&gt;


&lt;!---------
---
class: left, middle


## Comparing imputation results to exome results

&lt;img src="img/results-exome-hits.png" width="800"&gt;
---------&gt;

&lt;!---------------- DISCUSSION ---------------&gt;

---
class: inverse, center, middle

# Discussion



---
class: left, middle

## Outlook for SSIMP

To impute summary statistics reliably the following is needed ‚òùÔ∏è



### <i class="fas  fa-globe " style="color:#E41A1C;"></i> <i class="fas  fa-globe " style="color:#377EB8;"></i> <i class="fas  fa-globe " style="color:#4DAF4A;"></i> <i class="fas  fa-globe " style="color:#984EA3;"></i> <i class="fas  fa-globe " style="color:#FF7F00;"></i> Large and diverse reference panels 

- to improve of LD matrix estimation
- to impute low-frequency variants 
- to impute cosmopolitan/mutliethnic GWASs

### <i class="fas  fa-wrench " style="color:gray;"></i> Software &amp; tooling 

- easy to use and maintained

### <i class="ion  ion-ios-cloud-download "></i> Publicly available GWAS data  

- the foundation of SSIMP




---
class: left, middle

## Conclusion <i class="fab  fa-pagelines " style="color:green;"></i>

With my work on summary statistic imputation I have helped to **link incomplete GWAS** summary statistics with **follow-up methods** (such as multi-trait analyses).

&lt;!---------
- **Improved SSIMP**: .small[improved imputation quality and optimised assembly of the LD matrix.]

- Quantified **performance &amp; utility of SSIMP**: .small[compared to genotype imputation, identified groups of genetic variants that are hard to impute, and demonstrated in a case study the utility of summary statistic imputation.] 
-----&gt;

---
class: inverse, center, middle

.big[&lt;font face="Yanone Kaffeesatz"&gt; Thank you! &lt;/font&gt;] &lt;!------<i class="fas  fa-smile "></i> ----------&gt;

Slides: [https://sinarueeger.github.io/publicdefense/slides#1](https://sinarueeger.github.io/publicdefense/slides#1)

Source code: [https://github.com/sinarueeger/publicdefense/](https://github.com/sinarueeger/publicdefense/)
    </textarea>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="https://platform.twitter.com/widgets.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function() {
  var d = document, s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})();</script>

<script>
(function() {
  var i, text, code, codes = document.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
})();
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
