---
title: "Integrative statistical analysis of -omics and GWAS data"
subtitle: "Public defense"
author: "Sina RÃ¼eger"
date: "21 September 2018" ##  (updated: `r Sys.Date()`)
output:
  xaringan::moon_reader:
    css: ["default-fonts", "default", "theme-public-defense.css"]
    lib_dir: libs
    nature:
      highlightStyle: github
      highlightLines: true
      countIncrementalSlides: false
      beforeInit: "https://platform.twitter.com/widgets.js"
      titleSlideClass: [top, right]
---

```{r setup, include=FALSE}
options(htmltools.dir.version = FALSE)
library(tidyverse)
theme_set(theme_bw())

knitr::opts_chunk$set(
  tidy = TRUE, 
  echo = FALSE
)

```

<!------- emoji `r emo::ji("open_mouth")`---------->
<!------- icons `r icon::fa_r_project(colour = "#88398a")`---------->
<!------- icons `r icon::ii_heart()` ---------->


---
class: inverse, left, middle

# Outline

- Introduction

- Results

- Discussion 

---
class: inverse, left, middle

# Outline

- Introduction (= explaining the title)

- Results (= what I did during the PhD)

- Discussion (= did it work? & outlook)


<!---------------- INTRODUCTION --------------->
---
class: inverse, center, middle

# Introduction

---
class: center, middle

## The goal to keep in mind: Cornerstones of Medicine

<img src="img/cornerstones.png" width="400">


<!---- Data analysis -------->
---
class: center, middle

## 1. What is "data analysis" and "statistics"?

---
class: left, middle

### Summarise data

#### Example: Distribution of human height in 100 individuals

```{r, fig.width = 6, fig.height = 3}

set.seed(3)
height <- rnorm(100, 164, sd = 8)
av <- mean(height)
dat <- data.frame(Height = height, y= 1)
ggplot(data = NULL) + geom_point(aes(Height, y), data = dat, size = I(3), shape = 1) + ggtitle("Human height.") +  theme(axis.title.y=element_blank(), axis.text.y=element_blank(), axis.ticks.y=element_blank()) + ylim(c(0.95, 1.05))

```



---
class: left, middle

### Summarise data

#### Example: Distribution of human height in 100 individuals

```{r, fig.width = 6, fig.height = 3}

set.seed(3)
height <- rnorm(100, 164, sd = 8)
av <- mean(height)
dat <- data.frame(Height = height, y= 1)
ggplot(data = NULL) + geom_point(aes(Height, y), data = dat, size = I(3), shape = 1) + ggtitle("Human height.") + geom_point(aes(x = av, y = 1), color = I("red"), size = I(3)) +  theme(axis.title.y=element_blank(), axis.text.y=element_blank(), axis.ticks.y=element_blank()) + geom_text(aes(x = av, y= 1.01, label = "Average"), color = I("red")) + ylim(c(0.95, 1.05))

```


---
class: left, middle

### Identify relationships between A and B

#### Example: Relationship between age, sex and LDL (low-density lipoprotein)

.small[From: Freedman et al. (2004)]

<img src="img/age_LDL.png" width="200"> <!----- <img src="img/age_HDL.png" width="200"> ------->



---
class: left, middle

### Identifying patterns

#### Example: Genes mirror geography within Europe

<img src="img/european-map.png" width="300">

.small[From: Novembre et al. (2008)]


---
class: left, middle

### Medical scores to assess patients

#### Example: APGAR score

Test done in newborns 1-5 minutes after birth. 

1. Skin color
2. Pulse rate
3. Reflex
4. Activity
5. Respiratory

<img src="img/title-apgar.png" width="400">

.small[From: Apgar (1953)]


---
class: left, middle

## What is the purpose of statistical methods? `r icon::fa_chart_line()`

1. By summarising data we can `r icon::fa_caret_down()` distill knowledge.

1. Helps us to understand the world and the complex systems we life in.

1. How data analysis works: 
    - have a problem
    - define a question
    - collect data
    - answer the question (limited to the data used)


---
class: left, middle

## Central principles of data analysis

- Data represents oftentimes an **approximation** for something we cannot measure `r icon::fa_arrow_right()` *Use of BMI as a proxy for obesity*.

- The more data records we have, the **more precise** we can quantify something  `r icon::fa_arrow_right()` *Relationship between sample size and the standard error*.

- Some questions are harder to answer than others: 
  - a **relationship between X and Y** is easier to quantify than showing that X is causing Y `r icon::fa_arrow_right()` * Correlation â‰  Causation*.
  - it is **impossible to prove** something, but we can give evidence that something is more likely `r icon::fa_arrow_right()` *aka. Hypothesis testing*.

---
class: left, middle

## Keep in mind `r icon::fa_hand_pointer()`

- Linear regression: estimate the effect of a predictor on an outcome. 
- Increasing sample size helps to estimate precisely. 

<!---- Biology -------->

---
class: center, middle

## 2. Bits of biology 

or: what is *-omics*?

---
class: left, middle

## DNA is the blueprint of human life

- we are originally just one cell with dna from mum and dna from dad

- DNA cnanot be changed easily

- A, C, G, T: SNP

- SNPs of famuous ppl

- LD, gummy bears

- complex traits: many genes are affecting... 

---
class: left, middle

## Layers

- RNA, ... 

<img src="img/cascade.pdf" width="400">

- basket of environment


---
class: left, middle

## Why are we interesed in DNA? 

- disecting genetic from environmental effects

- knowing risk per person to develop a disease

<img src="img/gwas_rationale.pdf" width="400">



---
class: left, middle

## How is DNA measures?

<img src="img/genome.pdf" width="400">



---
class: left, middle

## Keep in mind `r icon::fa_hand_pointer()`

- DNA cannot be changed
- LD
- SNP

<!---- GWAS  -------->
<!--- combine DA and biology -------->
---
class: center, middle

## 3. GWAS (genome-wide association study)

---
class: left, middle

## Relationship between DNA and trait/disease

> Illustration

---
class: left, middle

## Relationship between DNA and trait/disease

> Illustration of 1 SNP

> Manhatten plot


---
class: left, middle

## TOP SNPs in height

- Increase height by... cm
- Top player basketball

---
class: left, middle

## Large sample sizes needed

<img src="img/maf_effectsize.pdf" width="400">



- Solution \#1: exploit LD and use cheap chips instead of pricy sequencing

- Solution \#2: Build consortium

---
class: left, middle

## How GWASs work 

- Large sample sizes needed > consortium

- Meta analsysis

- Make results public

---
class: left, middle

## Example GWAS

of a latest paper

---
class: left, middle

## Keep in mind `r icon::fa_hand_pointer()`

- Large sample sizes required: consortium!

- Recycling GWAS summary stats for other methods



<!---- Missing data -------->
---
class: center, middle

## 4. Missing data

---
class: left, middle

## Analogy

- How to infer data that is missing

- Analogy: Hiking Map

---
class: left, middle

## Basic principle


<img src="img/genome_imputation.pdf" width="400">


Reference panel html


<!---- SSIMP -------->
---
class: center, middle

## 3. Summary statistic imputation (SSIMP)

---
class: left, middle

## How SSimp works

<img src="img/summarystats_comic.pdf" width="400">


<!---------------- RESULTS --------------->
---
class: inverse, center, middle

# Results


---
class: left, middle
# Output


1. `r icon::fa_arrow_right()` Evaluation and application of summary statistic imputation `r icon::fa_arrow_left()`
    - <small>*Evaluation and application of summary statistic imputation to discover new height- associated loci*  (RÃ¼eger, McDaid, and Kutalik 2018)</small>

1. Improving summary statistic imputation for mixed populations
    - <small>*Improved imputation of summary statistics for mixed populations*  (RÃ¼eger, McDaid, and Kutalik 2017, not peer-reviewed)</small>

1. Applications of summary statistic imputation
    - <small>*Rare and low-frequency coding variants alter human adult height* (Marouli et al. 2017)</small>
    - <small>*Bayesian association scan reveals loci associated with human lifespan and linked biomarkers* (McDaid et al. 2017)/small>

1. Contributions to research in infectious diseases
    - <small>*Impact of com- mon risk factors of fibrosis progression in chronic hepatitis C* (RÃ¼eger et al. 2015)</small>
    - <small>*A systematic review and meta-analysis of HCV clearance* (Gauthiez et al. 2017)</small>
    - <small>*IL1B and DEFB1 polymorphisms increase susceptibility to inva- sive mold infection after solid-organ transplantation* (WÃ³jtowicz et al. 2015)</small>
    - <small>*PTX3 Polymorphisms and Invasive Mold Infections after Solid Organ Transplant* (WÃ³jtowicz et al. 2015)</small>

1. Minor contributions to other publications

---
class: left, middle
# Evaluation and application of summary statistic imputation (SSIMP)

<img src="img/rueeger-etal-2018-fp.pdf" width="600">

---
class: left, middle

## Goals

1. Improve imputation quality & investigate parameters of SSIMP.

1. Compare summary statistic imputation to genotype imputation.

1. Test the utility of summary statistic imputation on a real case on human height data.


---
class: left, middle

## How to test new methods?

- ðŸ™ˆ 

- Blind resutls!

---
class: left, middle

## 1) 
F1
F4/F5

---
class: left, middle

## 2) 
F2
Fig8
S6_Fig
S8_Fig


<!---------------- DISCUSSION --------------->

---
class: inverse, center, middle

# Discussion



---
class: left, middle

## What needs to be improved for SSIMP

To impute summary statistics the following is needed
- large and diverse reference panels >> improvement of LD matrix estimation.
- software/tools >> easy to use and maintained
- publicly available GWAS data >> the input of SSIMP
- figure out how to incorporate SSIMP into cohort pipeline


---
class: left, middle

## What is needed for GWASs

Foundation of SSIMP 

- Quality of GWAS
- More -omics data / deep phenotyinging
- Mutliethnic GWASs
- Public data: 
- Large & diverse reference panels


---
class: left, middle

## My contribution

With my work on summary statistic imputation I have helped to improve linking incomplete GWAS summary statistics with follow-up methods.

- SSIMP: Bridging incomplete GWAS summary statistics with follow up methods (For example: Multi-trait analyses (genetic correlation, pleiotropy))
- Improving SSIMP (improved imputation quality and optimised assembly of the LD matrix)
- Quantified how well SSIMP works (compared to genotype imputation, identified groups of genetic variants that are hard to impute, and demonstrated in a case study the util- ity of summary statistic imputation). 

